### PR TITLE
Ch9-13 Final check

### PR DESCRIPTION
**Pull request by trevorcampbell**

### CODE REVIEW

File: inference.Rmd [hunk 1]
```diff
@@ -39,7 +39,7 @@ By the end of the chapter, readers will be able to do the following:
 
 * Describe real-world examples of questions that can be answered with statistical inference.
 * Define common population parameters (e.g., mean, proportion, standard deviation) that are often estimated using sampled data, and estimate these from a sample.
-* Define the following statistical sampling terms (population, sample, population parameter, point estimate, sampling distribution).
+* Define the following statistical sampling terms: population, sample, population parameter, point estimate, and sampling distribution.
 * Explain the difference between a population parameter and a sample point estimate.
 * Use R to draw random samples from a finite population.
 * Use R to create a sampling distribution from a finite population.
```
Comment (leem44) [inline inference.Rmd:45]: The question comes after Figure 10.1, not a big deal but looks a little odd

File: jupyter.Rmd [hunk 2]
```diff
@@ -104,8 +104,7 @@ To run a code cell independently, the cell needs to first be activated. This
 is done by clicking on it with the cursor. Jupyter will indicate a cell has been
 activated by highlighting it with a blue rectangle to its left. After the cell
 has been activated (Figure \@ref(fig:activate-and-run-button)), the cell can be run by either pressing the **Run** (`r fa("play", height = "11px")`) 
-button in the toolbar, or by using a keyboard shortcut of 
-`Shift + Enter`.
+button in the toolbar, or by using the keyboard shortcut `Shift + Enter`.
```
Comment (leem44) [inline jupyter.Rmd:107]: the list on pg 350-351 has one bullet spilling onto the next pg

File: clustering.Rmd [hunk 3]
```diff
@@ -903,8 +902,8 @@ standardized_data
 
 To perform K-means clustering in R, we use the `kmeans` function. \index{K-means!kmeans function} It takes at
 least two arguments: the data frame containing the data you wish to cluster,
-and K, the number of clusters (here we choose K = 3). Note that since the K-means
-algorithm uses a random initialization of assignments, but since we set the random seed
+and K, the number of clusters (here we choose K = 3). Note that the K-means
```
[the remark this answers is not in the document]
Comment (trevorcampbell) [reply clustering.Rmd:905]: done

File: inference.Rmd (hunk 1 above)
Comment (trevorcampbell) [reply inference.Rmd:45]: fixed

File: jupyter.Rmd (hunk 2 above)
Comment (trevorcampbell) [reply jupyter.Rmd:107]: done

**Pull request merged**